server/resource/v1/history_resource.go

```diff
@@ -33,16 +33,17 @@ import (
 type HistoryResource struct {
 }
 
-//GetRevisionsByLabelID search key only by label
-func (r *HistoryResource) GetRevisionsByLabelID(context *restful.Context) {
+//GetRevisions search key only by label
+func (r *HistoryResource) GetRevisions(context *restful.Context) {
 	var err error
 	labelID := context.ReadPathParameter("label_id")
 	if labelID == "" {
 		openlogging.Debug("label id is null")
 		WriteErrResponse(context, http.StatusForbidden, "label_id must not be empty", common.ContentTypeText)
 		return
 	}
-	revisions, err := service.HistoryService.GetHistoryByLabelID(context.Ctx, labelID)
```

Comment:
Are there some condition parameters will be added in querying history resources later？For example time spans.